nixos/modules/system/boot/systemd.nix

```diff
@@ -503,16 +503,16 @@ in
     environment.systemPackages = [ cfg.package ];
 
     environment.etc = let
-      # generate contents for /etc/systemd/system-${type} from attrset of links and packages
-      hooks = type: links: pkgs.runCommand "system-${type}" {
+      # generate contents for /etc/systemd/${scope}-${type} from attrset of links and packages
```

Comment:
Hmmh, `scope` is a bit of a [misleading name[(https://www.freedesktop.org/software/systemd/man/latest/systemd.scope.htm), especially in systemd context.

https://www.freedesktop.org/software/systemd/man/latest/systemd.unit.html#Unit%20File%20Load%20Path distinguishes between "user mode" and "system mode", maybe call this "mode", or just make a single dir/path argument instead of `${scope}-${type}` (my preference)?

Reply:
Done according to your preference :smile: